css/style.css

```diff
@@ -6,6 +6,160 @@ body {
     -webkit-font-smoothing: antialiased;
 }
 
+/* This is the header part */
+
+header {
+    display: flex;
+    flex-direction: row;
+    margin: 1rem;
+    justify-content: center;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.logo {
+    width: 3rem;
+    height: 3rem;
+}
+
+#nav-item1 {
+    font-weight: bold;
+}
+
+.nav-item {
+    margin: 1rem;
+    text-align: center;
+    width: auto;
+
+}
+
+.nav-item:hover {
+    color: #E0633A;
+}
+
+.top-navbar {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin: 0 3rem;
+}
+
+/* This is the hero section */
+
+.hero {
+    background-image: url("../img/first-background.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 39rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+h1 {
+    color: white;
+    font-weight: lighter;
+    font-size: 3.25rem;
+    margin: 0;
+}
+
+.hero-text {
+    color: white;
+    margin: 1rem;
+    font-weight: lighter;
+    font-size: 2rem;
+}
+
+.learn-btn {
+    background-color: #E0633A;
+    font-size: 1.5rem;
+    border: none;
+    color: white;
+    padding: 0.75rem 1.5rem;
+    border-radius: 0.25rem;
+    margin: 2rem;
+}
+
+/* This is the icon section  */
+
+.product-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+h2 {
+    font-weight: lighter;
+    font-size: 2.5rem;
+    margin-top: 5rem;
+}
+
+
+.product-nav {
+    display: flex;
```

Comment:
Hi Afsha, such great job you've done, may i suggest implementing the DRY principle of not repeating code, so you could group all your flex items together.